packages/office-ui-fabric-react/src/utilities/dragdrop/interfaces.ts

```diff
@@ -18,6 +18,7 @@ export interface IDragDropEvents {
   onDrop?: (item?: any, event?: DragEvent) => void;
   onDragStart?: (item?: any, itemIndex?: number, selectedItems?: any[], event?: MouseEvent) => void;
   onDragEnd?: (item?: any, event?: DragEvent) => void;
+  onDragOver?: (item?: any, event?: DragEvent) => void;
```

Comment:
>onDragOver?: (item?: any, event?: DragEvent) => void; [](start = 2, length = 53)

Do we have to expose this in IDragDropEvents interface?

Reply:
Yes, as this is the only event that gets triggered while dragging on a cell, and the drop hint status needs to be changed while crossing the 50% of cell width